docs/data/joy/components/alert/AlertColors.tsx

```diff
@@ -20,7 +20,7 @@ export default function AlertColors() {
       <Box
         sx={{
           display: 'grid',
-          gridTemplateColumns: 'repeat(auto-fit, minmax(150px, 1fr))',
+          gridTemplateColumns: 'repeat(auto-fit, minmax(150px, 400px))',
```

Comment:
Are you sure?

Reply:
I thought I was! 😅 I'm stumped. What's a better way to do it?

Reply:
I feel. that we don't need CSS Grid in this case, a <Stack> would do the job.

Reply:
true.

Reply:
Ah that makes sense. The CSS Grid does feel a bit overcomplicated here. I'll try refactoring it.